ch5-catdir/directory.h

```diff
@@ -0,0 +1,35 @@
+/***********************************************
+ * directory.h - Definision of class Directory *
```

Comment:
```suggestion
 * directory.h - Definition of class Directory *
```

Reply:
Or remove the file comment altogether... :)

Reply:
Yes, I removed them.